pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/DSMServer.java

```diff
@@ -1086,6 +1087,13 @@ private void setupRouteGenericErrorHandlers() {
             response.status(500);
             response.body(exception.getMessage());
         });
+        // todo not sure if this will remain in this ticket or not
+        exception(UnsafeDeleteError.class, (exception, request, response) -> {
+            logger.error("DSM is unable to delete the object {}",  exception.toString());
+            exception.printStackTrace();
+            response.status(500);
+            response.body(exception.getMessage());
+        });
```

Comment:
A way to respond to frontend for in case the object they want to delete is in the a clinical order

Reply:
I'm not sure we want to do anything in response to this-if someone tries to do something that the system should prevent, and the system prevents it, there's no need for us to be alerted to it, although the user should receive some sort of indication of what happened.